internal/server/v1beta1/server.go

```diff
@@ -28,23 +28,27 @@ type APIServer struct {
 
 var errMissingUserInfo = errors.New("missing user information")
 
-func NewAPIServer(
-	logger log.Logger,
-	assetService AssetService,
-	starService StarService,
-	discussionService DiscussionService,
-	tagService TagService,
-	tagTemplateService TagTemplateService,
-	userService UserService,
-) *APIServer {
+type APIServerDeps struct {
+	Asset       AssetService
+	Star        StarService
+	Discussion  DiscussionService
+	Tag         TagService
+	TagTemplate TagTemplateService
+	User        UserService
+	Logger      log.Logger
+	StatsD      StatsDClient
```

Comment:
I know it is shorter to write, but i think better to use the original or shortened name of it to have less ambiguity,

For example,  instead of `Asset`, better to use `AssetService` or `AssetSvc`, to prevent people accidentally thinking to passing an Asset to this when writing the code